src/fastoad/model_base/flight_point.py

```diff
@@ -59,78 +73,127 @@ class FlightPoint:
-    drag: float = None  #: Aircraft drag in Newtons.
-    thrust: float = None  #: Thrust in Newtons.
-    thrust_rate: float = None  #: Thrust rate (between 0. and 1.)
+    CD: float = field(default=None, metadata={FIELD_DESCRIPTOR: _FieldDescriptor(unit="-")})
+
+    #: Aircraft lift in Newtons
+    lift: float = field(default=None, metadata={FIELD_DESCRIPTOR: _FieldDescriptor(unit="N")})
+
+    #: Aircraft drag in Newtons.
+    drag: float = field(default=None, metadata={FIELD_DESCRIPTOR: _FieldDescriptor(unit="N")})
+
+    #: Thrust in Newtons.
+    thrust: float = field(default=None, metadata={FIELD_DESCRIPTOR: _FieldDescriptor(unit="N")})
+
+    #: Thrust rate (between 0. and 1.)
+    thrust_rate: float = field(
+        default=None, metadata={FIELD_DESCRIPTOR: _FieldDescriptor(unit="-")}
+    )
 
     #: If True, propulsion should match the thrust value.
     #: If False, propulsion should match thrust rate.
-    thrust_is_regulated: bool = None
-
-    sfc: float = None  #: Specific Fuel Consumption in kg/N/s.
-    slope_angle: float = None  #: Slope angle in radians.
-    acceleration: float = None  #: Acceleration value in m/s**2.
-    alpha: float = 0.0  #: angle of attack in radians
-    slope_angle_derivative: float = None  #: slope angle derivative in rad/s
-    name: str = None  #: Name of current phase.
-
-    _units = dict(
-        time="s",
-        altitude="m",
-        ground_distance="m",
-        mass="kg",
-        consumed_fuel="kg",
-        true_airspeed="m/s",
-        equivalent_airspeed="m/s",
-        mach="-",
-        CL="-",
-        CD="-",
-        lift="N",
-        drag="N",
-        thrust="N",
-        thrust_rate="-",
-        sfc="kg/N/s",
-        slope_angle="rad",
-        acceleration="m/s**2",
-        alpha="rad",
-        slope_angle_derivative="rad/s",
+    thrust_is_regulated: bool = field(
+        default=None, metadata={FIELD_DESCRIPTOR: _FieldDescriptor(unit="-")}
+    )
+
+    #: Specific Fuel Consumption in kg/N/s.
+    sfc: float = field(default=None, metadata={FIELD_DESCRIPTOR: _FieldDescriptor(unit="kg/N/s")})
+
+    #: Slope angle in radians.
+    slope_angle: float = field(
+        default=None, metadata={FIELD_DESCRIPTOR: _FieldDescriptor(unit="rad")}
+    )
+
+    #: Acceleration value in m/s**2.
+    acceleration: float = field(
+        default=None, metadata={FIELD_DESCRIPTOR: _FieldDescriptor(unit="m/s**2")}
+    )
+
+    #: angle of attack in radians
+    alpha: float = field(default=0.0, metadata={FIELD_DESCRIPTOR: _FieldDescriptor(unit="rad")})
+
+    #: slope angle derivative in rad/s
+    slope_angle_derivative: float = field(
+        default=None, metadata={FIELD_DESCRIPTOR: _FieldDescriptor(unit="rad/s")}
     )
 
+    #: Name of current phase.
+    name: str = field(default=None, metadata={FIELD_DESCRIPTOR: _FieldDescriptor()})
+
+    __field_descriptors = {}  # Will store field metadata when needed. Must be accessed through
```

Comment:
Must be accessed throught what ?

Reply:
Woops.... Fixed in ea410bf211517df3bd007729274aad531c67f9f1.